workbench/src/renderer/components/SetupTab/ArgInput/index.jsx

```diff
@@ -8,7 +8,10 @@ import Col from 'react-bootstrap/Col';
 import Row from 'react-bootstrap/Row';
 import InputGroup from 'react-bootstrap/InputGroup';
 import Modal from 'react-bootstrap/Modal';
-import { MdFolderOpen, MdInfo } from 'react-icons/md';
+import { MdFolderOpen, MdInfo, MdOpenInNew } from 'react-icons/md';
+import { shell } from 'electron';
+
+const baseUserguideURL = 'https://storage.googleapis.com/natcap-dev-build-artifacts/invest/emlys/3.9.1.post3250+g130227a76/userguide';
```

Comment:
So that you can try it out, I've left this pointing to a dev build that has the new section anchors. I'll change this to the standard `https://storage.googleapis.com/releases.naturalcapitalproject.org/invest-userguide/latest` before merging.

Reply:
@emlys Oh I almost forgot about this. How should we handle it? Do we need to update the UG to build the anchors before replacing this URL and merging?